modules/cnf-configuring-enhanced-log-filtering-for-linuxptp.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-configuring-enhanced-log-reduction-for-linuxptp_{context}"]
+= Configuring enhanced PTP log reduction
+
+Basic log reduction effectively filters out frequent logs, however if you want a periodic summary of those filtered logs use the enhanced log reduction feature. 
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+* Install the PTP Operator.
+
+.Procedure
+
+. Edit the `PtpConfig` CR:
++
+[source,terminal]
+----
+$ oc edit PtpConfig -n openshift-ptp
+----
+
+. In `spec.profile`, add the `ptpSettings.logReduce` specification and set the value to `enhanced`:
++
+[source,yaml]
+----
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: <ptp_config_name>
+  namespace: openshift-ptp
+...
+spec:
+  profile:
+  - name: "profile1"
+...
+    ptpSettings:
+      logReduce: "enhanced"
+----
+
+. Optional: Configure the interval for summary logs and a threshold in nanoseconds for the master offset logs. For example to set the interval to 60 seconds and the threshold to 100 nanoseconds, edit the `spec.profile`, add the `ptpSettings.logReduce` specification and set the value to `enhanced`
```

Comment:
Is there any way to avoid saying "master" in this case? Also, the value is not just `enhanced` so I'd expand it as it is in the YAML.
```suggestion
. Optional: Configure the interval for summary logs and a threshold in nanoseconds for the master offset logs. For example, to set the interval to 60 seconds and the threshold to 100 nanoseconds, add the `ptpSettings.logReduce` specification in the `spec.profile` section and set the value to `enhanced 60s 100`
```